test/topology/conftest.py

```diff
@@ -116,15 +117,15 @@ def cluster_con(hosts: List[IPAddress], port: int, use_ssl: bool):
 
 
 @pytest.fixture(scope="session")
-async def manager_internal(event_loop, request):
+async def manager_internal(event_loop, request, thread_pool):
```

Comment:
Let's move the thread_pool fixture closed to where it's used (i.e. before `manager_internal`).

Even better, since you're storing the thread_pool in ManagerClient now, we shouldn't need a separate fixture at all.
Can we just construct the thread pool inside `manager_internal`?

Reply:
Moved it to the constructor of `ManagerClient`